packages/replay/src/replay.ts

```diff
@@ -947,6 +947,11 @@ export class ReplayContainer implements ReplayContainerInterface {
 
     const replayEvent = await getReplayEvent({ scope, client, replayId, event: baseEvent });
 
+    if (!replayEvent) {
```

Comment:
We actually did not handle this case before (I guess types _can be_ helpful after all xD), which can be the case when using an event processor.